### DISCUSSION TITLE
Authelia redirect after login

### DISCUSSION THREAD

**Discussion by msr-financial-com** [authelia/authelia#5694] [Q&A] [answered]
When i am trying to access my kibana ingress with this link for example:
https://kibana.com/s/infrastructure/app/discover#/?_g=(filters:!(),refreshInterval:(pause:!t,value:0),time:(from:%272023-07-18T14:34:57.704Z%27,to:%272023-07-18T14:45:16.226Z%27))&_a=(columns:!(path,status,method,duration,bytes_sent,request_length),filters:!((%27$state%27:(store:appState),meta:(alias:!n,disabled:!f,index:%foo%27,key:vhost,negate:!f,params:(query:%27dashboard.com%20%27),type:phrase),query:(match_phrase:(vhost:%27dashboard.com%20%27))),(%27$state%27:(store:appState),meta:(alias:!n,disabled:!f,index:%foo%27,key:kubernetes_labels_app,negate:!f,params:(query:ingress-nginx),type:phrase),query:(match_phrase:(kubernetes_labels_app:ingress-nginx))),(%27$state%27:(store:appState),meta:(alias:!n,disabled:!f,index:%foo%27,key:method,negate:!f,params:(query:PUT),type:phrase),query:(match_phrase:(method:PUT))),(%27$state%27:(store:appState),meta:(alias:!n,disabled:!f,index:%foo%27,key:path,negate:!f,params:(query:%2Fmm5%2Fauth%2Fapi%2Fv1%2Fuserdata%2Fwidgets%2FviewChart),type:phrase),query:(match_phrase:(path:%2Fmm5%2Fauth%2Fapi%2Fv1%2Fuserdata%2Fwidgets%2FViewChart)))),index:%foo%27,interval:auto,query:(language:kuery,query:%27path:%2Fmt5%2Fauth%2F%27),sort:!(!(%27@timestamp%27,desc)))

This ingress is secured with authelia. After login i am getting redirected to:
https://kibana.com/s/it-infrastructure/app/discover

All the parts after "discover" are removed. I have to first login and then access the original link again, then i am landing on the correct page.

Is there an option i can add to the ingress or authelia config, so i am getting redirected to the original url after login?

**Comment by james-d-elliott** [comment, marked as answer]
The values AFTER the `#` (the document [fragment](https://developer.mozilla.org/en-US/docs/Web/API/DocumentFragment)) are never sent to the server by the browser, so unfortunately there is no way.
